tools/toolchains/arm.py

```diff
@@ -66,6 +66,9 @@ def __init__(self, target, notify=None, macros=None,
         self.asm = [main_cc] + self.flags['common'] + self.flags['asm']
         self.cc = [main_cc] + self.flags['common'] + self.flags['c']
         self.cppc = [main_cc] + self.flags['common'] + self.flags['c'] + self.flags['cxx']
+        if target.name in ["REALTEK_RTL8195AM"]:
```

Comment:
@theotherjimmy We think enum_is_int will added only when target.name is REALTEK_RTL8195AM, So ... may I ask what kind of blocking is this?